pypeflow/pwatcher_bridge.py

```diff
@@ -30,15 +31,19 @@ def PypeProcWatcherWorkflow(
         URL = None,
         job_type='local',
         job_queue='UNSPECIFIED_QUEUE',
+        watcher_type='fs_based',
+        watcher_directory='mypwatcher',
         **attributes):
     """Factory for the workflow using our new
     filesystem process watcher.
     """
-    if job_type == 'string':
+    if job_type == 'string' or watcher_type == 'string':
```

Comment:
We will drop `job_type=string` soon. And we will shorten `watcher_type` to `pwatcher`.